yeat/tests/__init__.py

```diff
@@ -9,15 +9,13 @@
 
 import json
 import multiprocessing
-import os
 from pathlib import Path
-from pkg_resources import resource_filename
+from importlib.resources import files
 
 
 def data_file(path):
-    pathparts = path.split("/")
-    relpath = os.path.join("tests", "data", *pathparts)
-    return resource_filename("yeat", relpath)
+    pkg_path = files("yeat") / "tests" / "data" / path
+    return str(pkg_path)
```

Comment:
Started seeing deprecation warnings a few months ago with a different workflow, so switched to importlib.resources and haven't had problems since!

Reply:
Are you on the `pkg/paths` branch?

Reply:
> Are you on the `pkg/paths` branch?

Yes.

I tried, python v3.10 as well and no luck.

Reply:
I ran the test suite on another machine and I got no "implicit namespace package" warnings. Going to continue figuring out what's going on with my main system.

Reply:
FINALLY FOUND A SOLUTION: 🎉🎉🎉🎉🎉🎉🎉

`pip uninstall setuptools`

I must have installed it on my laptop when I was trying to get `metaMDBG` on my machine...